.gitignore

```diff
@@ -1,2 +1,5 @@
 target
 Cargo.lock
+
+*.swp
+*.swo
```

Comment:
I usually prefer to keep these ignores in my `.gitignore_global` since they're not project specific. Can you please remove these changes?